### PR TITLE
[Scene] Name Tags with Health Bar

### PR DESCRIPTION
**Pull request by UDXS**
I Have Added Name Tags using the names of each character. These are small semi-transparent bars. that appear under each character. The Top Gray Bar has their name and the bottom red bar has their health. The Health Text shows the new health values i've programmed including the max hp points the character can have and the current percentage that the character has. I have modified the Settings Menu to fit an option that lets you show the health as a percentage or as a fraction.

A Demo Image

### CODE REVIEW

**Comment by gunthergun** [comment]
Please add [scene]  in the tittle this lets the mods know that it changes the scene

**Comment by svmnotn** [comment]
also, you might want to fix the conflicts

**Comment by UDXS** [comment]
How do you check conflicts?

**Comment by Mizipzor** [comment]
Scene conflicts are _really_ hard to get right anyway. :/

**Comment by UDXS** [comment]
Awe:)

**Comment by UDXS** [comment]
I meant :(

**Comment by Mizipzor** [comment]
You can edit posts. ;)

**Comment by UDXS** [comment]
Not on mobile

**Comment by UDXS** [comment]
Also how do you add labels

**Comment by Mizipzor** [comment]
You cant yourself, only people with elevated permissions. I hate that part of Github.

File: Assets/Scripts/UI/NameTagUpdater.cs [hunk 1]
```diff
@@ -6,11 +6,6 @@ public class NameTagUpdater : MonoBehaviour {
     public Character character;
     public SettingHelper settings;
 	void Update () {
-		character.health = character.health -= 0.01f;
-		if (character.health < 0)
-		{
-			character.health = 1;
-		}
```
Comment (Mizipzor) [inline Assets/Scripts/UI/NameTagUpdater.cs]: Use spaces instead of tabs.

**Comment by UDXS** [comment]
I'm closing this so i can redo it 100% programmatically.

**Pull request closed without merging**